doc/GitCrashCourse.md

```diff
@@ -0,0 +1,543 @@
+<!--
+Copyright (c) 2020, 2020 IBM Corp. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at http://eclipse.org/legal/epl-2.0
+or the Apache License, Version 2.0 which accompanies this distribution
+and is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following Secondary
+Licenses when the conditions for such availability set forth in the
+Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+version 2 with the GNU Classpath Exception [1] and GNU General Public
+License, version 2 with the OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Git crash course
+
+Git is an open-source distributed version control system that has quickly
+become the most popular version control system for tracking code changes
+during software development. Git is considered a distributed version
+control system because every contributor to a project can have a complete
+copy of a repository and make changes independently. The contributors could
+then request to have their code changes reviewed and merged into the main
+project repository, making it possible for large open-source projects to
+have many contributors.
+
+You will need to be able to work with git to make code contributions to
+Eclipse OMR. This guide is meant to help you get started and understand
+the project's workflow using git and GitHub by walking you through the process of
+making your first contribution to the Eclipse OMR project hosted on GitHub.
+It is by no means a complete guide for all you can do with git -- there
+are plenty of excellent online resources for that.
+
+# Setting up git and GitHub
+
+Although there are many git GUIs available for download, this guide will
```

Comment:
This doesn't seem to add much - why not just say this guide will focus on using the git cli - you may choose to use a GUI in addition / instead if you so choose.